arch/src/riscv64/fdt.rs

```diff
@@ -119,7 +119,7 @@ fn create_cpu_nodes(fdt: &mut FdtWriter, num_cpus: u32) -> FdtWriterResult<()> {
     fdt.property_u32("timebase-frequency", timebase_frequency)?;
 
     for cpu_index in 0..num_cpus {
```

Comment:
This commit message for this clippy change should have been "arch:" since it only touched the one file - but i'm not going to waste compute resources and your time asking you to respin for that.

Reply:
I think what we have works fine.